### ISSUE TITLE
LND Binary Library Unsuccessfully Fetching LND 

### ISSUE THREAD

**Issue by lieteau2** [LN-Zap/zap-desktop#1002]
Going through advanced guide for Zap Installation, when spinning up Zap via yarn dev I was getting the following error; 

<img width="1280" alt="screen shot 2018-12-03 at 9 50 22 am" src="https://user-images.githubusercontent.com/45567325/49384738-46c21880-f6e1-11e8-971b-bda837392a6c.png">

When running `yarn fetch-lnd` I received the following output error

```
info lnd-binary Downloading https://github.com/LN-Zap/lnd/releases/download/v0.5.1-beta/lnd-darwin-amd64-v0.5.1-beta.tar.gz
ERR! lnd-binary Checksum did not match
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  Error verifying checksum of downloaded file { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
info lnd-binary Downloading https://github.com/LN-Zap/lnd/releases/download/v0.5.1-beta/lnd-darwin-386-v0.5.1-beta.tar.gz
ERR! lnd-binary Checksum did not match
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  Error verifying checksum of downloaded file { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
info lnd-binary Downloading https://github.com/LN-Zap/lnd/releases/download/v0.5.1-beta/lnd-windows-amd64-v0.5.1-beta.zip
ERR! lnd-binary Checksum did not match
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  Error verifying checksum of downloaded file { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
info lnd-binary Downloading https://github.com/LN-Zap/lnd/releases/download/v0.5.1-beta/lnd-windows-386-v0.5.1-beta.zip
ERR! lnd-binary Checksum did not match
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  Error verifying checksum of downloaded file { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
info lnd-binary Downloading https://github.com/LN-Zap/lnd/releases/download/v0.5.1-beta/lnd-linux-amd64-v0.5.1-beta.tar.gz
ERR! lnd-binary Checksum did not match
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  Error verifying checksum of downloaded file { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
info lnd-binary Downloading https://github.com/LN-Zap/lnd/releases/download/v0.5.1-beta/lnd-linux-386-v0.5.1-beta.tar.gz
ERR! lnd-binary Checksum did not match
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  Error verifying checksum of downloaded file { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
ERR! lnd-binary Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary  { Error: Checksum did not match
ERR! lnd-binary     at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)
ERR! lnd-binary     at process._tickCallback (internal/process/next_tick.js:68:7)
ERR! lnd-binary   stack:
ERR! lnd-binary    'Error: Checksum did not match\n    at _hasha2.default.fromFile.then.hash (/Users/dylan/Zap/zap-desktop/node_modules/lnd-binary/src/lib/verify.js:41:29)\n    at process._tickCallback (internal/process/next_tick.js:68:7)' }
```

**Comment by mrfelton** [comment]
@JimmyMow I think this is due to the fact that I cut a new version of the 0.5.1 binary today in order to ARM support.

Can you try clearing your npm cache please?

**Comment by lieteau2** [comment]
@mrfelton Issue resolved with PR #1003 

closing issue
